### PR TITLE
OP-1811: nctl nightly run

### PR DESCRIPTION
**Pull request by TomVasile**
Initial commit of nightly nctl cronjob.

### CODE REVIEW

**Comment by TomVasile** [comment]
@momipsl Included you just for fyi to the change to the sync scenario that we discussed.

**Review by momipsl — APPROVED** [review]

**Comment by goral09** [comment]
I can't see it anywhere in the CI https://drone-auto.casperlabs.io/CasperLabs/casper-node/4499/1/1 . Was it expected to run?

File: ci/nightly-test.sh [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+echo "Sleeping 60 to allow network startup"
+sleep 60
+
+# Switch to scenarios
+pushd $SCENARIOS_DIR
+source sync_test.sh node=6 timeout=500
+
+# Switch back and teardown
+popd
+nctl-assets-teardown
```
Comment (goral09) [inline ci/nightly-test.sh:34]: If we ever want to run more IT tests like this, we probably will have to set up individual nctl assets (and networks) for each of them.

**Comment by TomVasile** [comment]
> I can't see it anywhere in the CI https://drone-auto.casperlabs.io/CasperLabs/casper-node/4499/1/1 . Was it expected to run?
> 
> ~The setup has changed, it won't work without the launcher anymore.~
> 
> OK, now I see it's part of the setup here.

yea wasnt supposed to run. It will get kicked off differently

File: ci/nightly-test.sh (hunk 1 above)
Comment (TomVasile) [reply ci/nightly-test.sh:34]: yea, figured I'd cross that bridge when we get there. This was just to get it going.

**Comment by goral09** [comment]
> > I can't see it anywhere in the CI [drone-auto.casperlabs.io/CasperLabs/casper-node/4499/1/1](https://drone-auto.casperlabs.io/CasperLabs/casper-node/4499/1/1) . Was it expected to run?
> > ~The setup has changed, it won't work without the launcher anymore.~
> > OK, now I see it's part of the setup here.
> 
> yea wasnt supposed to run. It will get kicked off differently

**Review by sacherjj — APPROVED** [review]

**Pull request merged**